lms/djangoapps/dashboard/tests/test_sysadmin.py

```diff
@@ -162,6 +162,71 @@ def test_staff_access(self):
         response = self.client.get(reverse('gitlogs'))
         self.assertTrue(response.status_code, 200)
 
+    def test_rename_user(self):
+
+        self._setstaff_login()
+        self.client.login(username=self.user.username, password='foo')
+
+        user1 = UserFactory.create(
+            username='test_rename_user_1',
+            email='test_rename_user_1@edx.org',
+            password='foo'
+        )
+        user2 = UserFactory.create(
+            username='test_rename_user_2',
```

Comment:
Similar to how we did this for the management command, could we set `user2`'s username w/ something like `username2=username1 + 'suffix'`? That helps ensure that we never accidentally break out test cases.